qgreenland/util/config/export.py

```diff
@@ -101,13 +105,14 @@ def export_config_csv(
                 "Subgroup": "/".join(layer_node.group_name_path[1:]),
                 "Layer Title": layer_cfg.title,
                 "Layer Description": layer_cfg.description,
-                "Vector or Raster": layer_type,
+                "Vector or Raster": vector_or_raster_data,
                 "Data Source Title": dataset_cfg.metadata.title,
                 "Data Source Abstract": dataset_cfg.metadata.abstract,
                 "Data Source Citation": dataset_cfg.metadata.citation.text,
                 "Data Source Citation URL": dataset_cfg.metadata.citation.url,
                 "Layer Size": naturalsize(layer_size_bytes),
                 "Layer Size Bytes": layer_size_bytes,
+                "Layer Data On Disk": layer_data_on_disk,
```

Comment:
Better name? `Layer Is Online-Only`? `Layer is remote`?

Reply:
"Internet-required layer?" I like this because it better matches our language in the layers panel (`Internet-required data`). That said, I feel "Internet-required data" should be simply "Internet required" in the layers panel.

Reply:
With or without the question mark doesn't matter much to me

Reply:
Going with `Internet Required?`.